python/mirascope/llm/clients/openai/responses/_utils/encode.py

```diff
@@ -86,6 +87,24 @@ def _encode_message(
     for part in message.content:
         if part.type == "text":
             result.append(EasyInputMessageParam(role=message.role, content=part.text))
+        elif part.type == "image":
+            image_url = (
+                part.source.url
+                if part.source.type == "url_image_source"
+                else f"data:{part.source.mime_type};base64,{part.source.data}"
+            )
+            result.append(
```

Comment:
Just to double check: when a single user message includes both text and an image, this change now emits two separate user input items. In the Responses API it is still a single turn, so the instruction text no longer travels with the image. Could we keep both parts inside one content list instead?